### ISSUE TITLE
VipsOperation: class "text" not found

### ISSUE THREAD

**Issue by yml** [h2non/bimg#63]
This might be related to #53 

I was trying to use Watermark and I got a crashed so I decide to run `bimg` test suite and got a very similar crash:

```
yml@garfield$ (git: master) go test github.com/h2non/bimg
--- FAIL: TestImageWatermark (0.06s)
        image_test.go:164: VipsOperation: class "text" not found

        image_test.go:169: Unsupported image format
        image_test.go:173: Image is not jpeg
--- FAIL: TestImageWatermarkNoReplicate (0.11s)
        image_test.go:195: VipsOperation: class "text" not found

        image_test.go:200: Unsupported image format
        image_test.go:204: Image is not jpeg
fatal error: unexpected signal during runtime execution
[signal 0xb code=0x1 addr=0x60 pc=0x7f8a4dbb0789]

runtime stack:
runtime.throw(0x5f13a0, 0x2a)
        /home/yml/go/src/runtime/panic.go:527 +0x90
runtime.sigpanic()
        /home/yml/go/src/runtime/sigpanic_unix.go:12 +0x5a

goroutine 48 [syscall, locked to thread]:
runtime.cgocall(0x403d20, 0xc820043d08, 0xc800000000)
        /home/yml/go/src/runtime/cgocall.go:120 +0x11b fp=0xc820043cd8 sp=0xc820043ca8
github.com/h2non/bimg._Cfunc_vips_colourspace_issupported_bridge(0x0, 0xc800000000)
        ??:0 +0x35 fp=0xc820043d08 sp=0xc820043cd8
github.com/h2non/bimg.vipsColourspaceIsSupported(0x0, 0xc820030080)
        /home/yml/gopath/src/github.com/h2non/bimg/vips.go:243 +0x21 fp=0xc820043d20 sp=0xc820043d08
github.com/h2non/bimg.vipsPreSave(0x0, 0xc820043df0, 0x0, 0x0, 0x0)
        /home/yml/gopath/src/github.com/h2non/bimg/vips.go:293 +0x88 fp=0xc820043d58 sp=0xc820043d20
github.com/h2non/bimg.vipsSave(0x0, 0x5f, 0x0, 0x0, 0x0, 0x16, 0x0, 0x0, 0x0, 0x0, ...)
        /home/yml/gopath/src/github.com/h2non/bimg/vips.go:307 +0x97 fp=0xc820043de8 sp=0xc820043d58
github.com/h2non/bimg.TestVipsWatermark(0xc820094e10)
        /home/yml/gopath/src/github.com/h2non/bimg/vips_test.go:90 +0x22b fp=0xc820043f68 sp=0xc820043de8
testing.tRunner(0xc820094e10, 0x891d50)
        /home/yml/go/src/testing/testing.go:456 +0x98 fp=0xc820043fa0 sp=0xc820043f68
runtime.goexit()
        /home/yml/go/src/runtime/asm_amd64.s:1696 +0x1 fp=0xc820043fa8 sp=0xc820043fa0
created by testing.RunTests
        /home/yml/go/src/testing/testing.go:561 +0x86d
```

As far as I can tell I am using libvips 8.1.1
```
yml@garfield$ (git: master) vipsthumbnail --vips-version
libvips 8.1.1-Fri Oct 30 17:21:28 CET 2015
```

Thank you for putting this great wrapper around libvips.

**Comment by h2non** [comment]
Please see: https://github.com/h2non/imaginary/issues/31#issuecomment-146011173

You've to compile `libvips` with `libpango` bindings: https://github.com/jcupitt/libvips#pangoft2.

If you use the `bash` [install script](https://github.com/h2non/bimg#libvips), all should work fine!

**Comment by yml** [comment]
@h2non 

Thank you very much for your quick answer that fixes my issue.

 Just in case someone else stumble on this  here it is how I fixed it:

* install pango dev files : `sudo aptitude install libpango1.0-dev`
* reinstall libvips using the [install script](https://github.com/h2non/bimg#libvips). You will need to tweak it before you run it.

  *  remove the line that checks for `vips` existence line 100 :+1:  `check_if_library_exists "vips" "$vips_version_minimum"`
  * I have also commented out in `install_libvips_from_source()` the 2 lines that delete the sources so I can run `make uninstall` in the future.
